proto/desmos/subspaces/v1beta1/models.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package desmos.subspaces.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/staging/subspaces/types";
+
+// Users represents a slice of users inside a specific subspace
+message Users {
```

Comment:
Yes I think so. I've added this type when I originally intend the `Admins` and `RegisteredUsers` as separated entities from the subspace and needed it because Protobuf won't let you marshal something that doesn't implement its `marshaler` methods.